src/main/kotlin/org/kamiblue/client/gui/hudgui/elements/misc/FPS.kt

```diff
@@ -55,7 +54,6 @@ object FPS : LabelHud(
         var min = 6969
         var max = 0
         for (value in longFps) {
-            if (value != 0) min = min(value, min)
```

Comment:
This is actually for avoiding min fps 0 to be displayed (First time joining a world)

Reply:
I noticed that. I'll add it back it